### PR TITLE
implement ctrl_key_map option

### PR DESCRIPTION
**Pull request by fredcallaway**
Note that we don't allow mapping keys which have critical functionality (e.g. remapping c-m would also affect enter).

### CODE REVIEW

**Comment by fredcallaway** [comment]
Fixes #404

File: radian/rutils.py [hunk 1]
```diff
@@ -78,6 +78,14 @@ def load_custom_key_bindings(*args):
     for m in esc_keymap:
         map_key(("escape", m["key"]), m["value"], mode=m["mode"] if "mode" in m else "r")
 
+    keymap = roption("radian.ctrl_key_map", [])
+    for m in keymap:
+        if m["key"] in "mihdc":
+            print("WARNING: Cannot remap c-" + m["key"] + ". Please remove this mapping from radian.ctrl_key_map in your .radian_profile")
```
Comment (randy3k) [inline radian/rutils.py:84]: The setting could be specified in various places. Could you rename .radian_profile to just "radian profile".

**Review by randy3k — COMMENTED** [review]
Thanks, just a little nitpick.

File: radian/rutils.py (hunk 1 above)
Comment (fredcallaway) [reply radian/rutils.py]: Done!

**Pull request merged**